docs/templates/templates.md

```diff
@@ -1,28 +1,30 @@
 The go template engine is used for YAGPDB's custom commands.
 This page aims to help you to get the most out of the custom commands feature.
 
+Put {{"{{"}}...{{"}}"}} around the names like that: `{{"{{"}}.User.Username{{"}}"}}`, for `<@.User.ID>` don't forget them aswell.
+
 Available template data:
 
 > type User
-* `{{.User.Username}}` Outputs the username
-* `{{.User.Avatar}}` Outputs the user avatar
-* `{{.User.ID}}` Outputs the user ID
-* `<@{{.User.ID}}>` Outputs a mention to the user with the user ID, doesn't output an ID
-* `{{.User.Discriminator}}` Outputs the user discriminator
-* `{{.User.Bot}}` Outputs true or false, if the user is a bot it will be `true` and if not then `false`
+* `.User.Username` Outputs the username
+* `.User.Avatar` Outputs the user avatar
+* `.User.ID` Outputs the user ID
+* `<@.User.ID>` Outputs a mention to the user with the user ID, doesn't output an ID
```

Comment:
Remove this one, and add the mention example somewhere else, like on the bottom with a "tips" or "snippets" section